api/src/main/java/org/pmiops/workbench/api/CohortsController.java

```diff
@@ -280,7 +280,7 @@ public Cohort apply(org.pmiops.workbench.db.model.Cohort cohort) {
     }
 
     MaterializeCohortResponse response = cohortMaterializationService.materializeCohort(
-        cohortReview, searchRequest, statusFilter, pageSize,
+        cohortReview, searchRequest, request.getFieldSet(), statusFilter, pageSize,
```

Comment:
Should be just pass the request as a param instead of request.getFieldSet() and request.getPageToken()?

Reply:
I changed it to take cohortReview, searchRequest, and cohortMaterializationRequest... the request now gets modified in CohortsController to set up defaults.